README.md

```diff
@@ -71,7 +71,8 @@
 ✓ [Certification Section](#certifications)  
 ✓ [Education Section](#education)  
 ✓ [Projects Section](#projects)  
-✓ [Blog Posts Section](#blog-posts)
+✓ [Blog Posts Section](#blog-posts)  
+✓ [Custom footer](#custom-footer)
```

Comment:
Let's not declare it as a feature. We can just show it on the example config of `Customization`.